frontend/src/aws-exports.js

```diff
@@ -9,7 +9,7 @@ const awsconfig = {
         domain: env.REACT_APP_AWS_DOMAIN || "prod-fam-user-pool-domain.auth.ca-central-1.amazoncognito.com",
         scope: ['openid'],
         redirectSignIn: `${window.location.origin}/dashboard`,
-        redirectSignOut: env.REACT_APP_REDIRECT_SIGN_OUT || "https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi?retnow=1&returl=https://dev.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/logout?redirect_uri=http://localhost:3000/",
+        redirectSignOut: env.REACT_APP_REDIRECT_SIGN_OUT || `https://logon${env.REACT_APP_ZONE.toLocaleLowerCase() === "prod"?'':'test'}7.gov.bc.ca/clp-cgi/logoff.cgi?retnow=1&returl=https://${env.REACT_APP_ZONE.toLocaleLowerCase()}.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/logout?redirect_uri=${window.location.origin}/`,
```

Comment:
For ease of hand off, maybe we should make this an external variable?  Then we'd just go with a simple default in aws-exports.js if that's not provided.

Reply:
yup, using REACT_APP_REDIRECT_SIGN_OUT could be one possible approach, but we would need to add that to the github variables then for each environment, and whenever we switch to local we would need to keep in mind, of adding a long url to the .env
like:
for DEV/LOCAL: 
https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi?retnow=1&returl=https://dev.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/logout?redirect_uri=http://localhost:3000/
for TEST in github var:
https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi?retnow=1&returl=https://test.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/logout?redirect_uri=https://nr-silva-test-frontend.apps.silver.devops.gov.bc.ca/
for PROD in github var:
https://logon7.gov.bc.ca/clp-cgi/logoff.cgi?retnow=1&returl=https://prod.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/logout?redirect_uri=https://nr-silva-prod-frontend.apps.silver.devops.gov.bc.ca/

Reply:
^ I like this.  :)